src/ScreenStages/GameScreen.cpp

```diff
@@ -2787,6 +2787,10 @@ void CGameScreen::OnKeyDown(const KeyEvent &ev)
     {
         return;
     }
+    else if (key == 113 && ctrlPressed)
```

Comment:
Is `CTRL+Q` default on original client? What happens with macros mapped to this combo?
Also, not that I removed windows specific code, we can fully use SDL. In this case, use `SDLK_q` instead `113`.

Reply:
yes in original client macro and this command will be excuse both but if we want to fix that return; ....

Reply:
Yes, I tested here and this is right. So use SDLK_q here and we're good! :)